superset/assets/javascripts/SqlLab/components/QuerySearch.jsx

```diff
@@ -185,16 +185,20 @@ class QuerySearch extends React.PureComponent {
             Search
           </Button>
         </div>
-        <QueryTable
-          columns={[
-            'state', 'db', 'user', 'date',
-            'progress', 'rows', 'sql', 'querylink',
-          ]}
-          onUserClicked={this.onUserClicked.bind(this)}
-          onDbClicked={this.onDbClicked.bind(this)}
-          queries={this.state.queriesArray}
-          actions={this.props.actions}
-        />
+        <div className="scrollbar-container">
```

Comment:
Now I use 95% in height to allow space for navbar header, which is pretty hacky. There's another solution here [https://github.com/airbnb/superset/pull/1611](url)